lib/internal/Magento/Framework/View/Layout/CacheKeyInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Please use PHP 7.0 type hinting syntax when possible

Reply:
This one I just don't know what is it about, can you be a little more specific please? Thanks!